frontend/src/scenes/onboarding/sdks/shared-snippets.tsx

```diff
@@ -0,0 +1,27 @@
+import { CodeSnippet, Language } from 'lib/components/CodeSnippet'
+import { Link } from 'lib/lemon-ui/Link'
+
+export function JSInstallSnippet(): JSX.Element {
+    return (
+        <CodeSnippet language={Language.Bash}>
+            {['npm install posthog-js', '# OR', 'yarn add posthog-js', '# OR', 'pnpm add posthog-js'].join('\n')}
+        </CodeSnippet>
+    )
+}
+
+export function SessionReplayFinalSteps(): JSX.Element {
+    return (
+        <>
+            <h3>Optional: Configure</h3>
+            <p>
+                Advanced users can add{' '}
+                <Link to="https://posthog.com/docs/libraries/js#config" target="_blank">
+                    configuration options
+                </Link>{' '}
+                to customize text masking, customize or disable event capturing, and more.
+            </p>
+            <h3>Create a recording</h3>
```

Comment:
Given this is related to both options I wonder if it should have some kind of separator before it
<img width="638" alt="Screenshot 2023-09-06 at 11 07 41" src="https://github.com/PostHog/posthog/assets/6685876/a70df53b-0191-4d66-8e8e-b6f29239f4a8">